recipe/meta.yaml

```diff
@@ -1,73 +1,74 @@
-{% set version = "0.4.1" %}
+{% set name = "dioptas" %}
+{% set version = "0.5.7" %}
 
 package:
-  name: dioptas
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/Dioptas/Dioptas/archive/{{ version }}.tar.gz
-  sha256: d2cf86417429f5092fc9db7d7a745706c027dafac1ee546c9a87eb5fd6d0d335
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dioptas-{{ version }}.tar.gz
+  sha256: cb3019da250aada747ed024d7ed473ddc23be10b22aa996fd076656d5045ca23
 
 build:
-  number: 1002
-  skip: true  # [win and py27]
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   entry_points:
-    - dioptas = dioptas:main
+    - dioptas = scripts:dioptas.main
+    # Requires sharedmem dependency not available in conda-forge
+    # - dioptas_batch = scripts:dioptas_batch.main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   build:
-    - {{ compiler('fortran') }}  
+    - {{ compiler('c') }}
   host:
     - python
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - cython
     - pip
-    - setuptools
-    - numpy
-    - scipy
-    - pyqt
-    - pyqtgraph
-    - lmfit
-    - fabio
-    - pyfai
-    - pycifrw
-    - future
-    - scikit-image
-    - qtpy
   run:
     - python
-    - {{ pin_compatible('numpy') }}
-    - scipy
-    - pyqt
-    - pyqtgraph
+    - cython
+    - future
+    - h5py
+    - hdf5plugin
     - lmfit
-    - fabio
-    - pyfai
+    - pandas
+    - psutil
     - pycifrw
-    - future
-    - scikit-image
+    - python-dateutil
+    - pyqt  # Use pyqt6 when available in conda-forge
```

Comment:
Using pyqt5 for now since Dioptas still works with it and pyqt6 is not available yet.